package.json

```diff
@@ -48,11 +48,11 @@
         "node": ">=10.0.0"
     },
     "dependencies": {
-        "ilib-loctool-webos-json-resource": "1.3.11"
+        "ilib-loctool-webos-json-resource": "1.3.11",
+        "loctool": "2.18.0"
```

Comment:
This should stay as a dev dependency or else you will get circular dependencies

Reply:
oh.. I see I re-updated it and webos-c plugin too.